.github/workflows/build-electron.yml

```diff
@@ -25,23 +29,33 @@ jobs:
       run: npm install
 
     - name: Fetch InVEST Binaries
-      shell: bash
       run: npm run fetch-invest
 
-    - name: Build & Release Electron application
-      uses: samuelmeuli/action-electron-builder@v1
-      if: ${{ always() }}
-      with:
-        build_script_name: build
-        args: -c.extraMetadata.main=build/main.js
-        github_token: ${{ secrets.github_token }}
-        # If the commit is tagged with a version (e.g. "v1.0.0"),
-        # release the app after building
-        release: ${{ startsWith(github.ref, 'refs/tags/v') }}
+    - name: NPM Install
+      run: npm install
```

Comment:
Looks like the npm install step is duplicated

Reply:
whoops!